static/lang/re-en.json

```diff
@@ -28,6 +28,7 @@
                 "HolyMace": "Holy Mace",
                 "Hoof": "Hoof",
                 "Horn": "Horn",
+                "Horns": "Horns",
```

Comment:
```suggestion
```
All other instances use Horn instead of Horns and given some discussions I've had I am inclined to think Paizo actually has a style guide on these and this one just slipped through